app/models/automated_checks/claim_verifiers/employment.rb

```diff
@@ -11,13 +11,18 @@ def initialize(claim:, admin_user: nil)
       end
 
       def perform
+        return unless required?
         return unless awaiting_task?
 
         no_data || no_match || matched
       end
 
       private
 
+      def required?
+        claim.eligibility.teacher_reference_number.present?
```

Comment:
not sure if this negatively impacts other policies?

Reply:
I think this should be ok, most of the other journeys require a TRN from the claimant to submit, IRP doesn't but it doesn't use this claim verifier